quilt/data.py

```diff
@@ -79,6 +72,16 @@ def __dir__(self):
         # https://mail.python.org/pipermail/python-ideas/2011-May/010321.html
         return sorted(set((dir(type(self)) + list(self.__dict__) + self._keys())))
 
+    def _is_group(self):
+        return isinstance(self._node, GroupNode)
+
+    def _is_df(self):
+        return not isinstance(self._node, GroupNode)
+
+    def _df(self):
```

Comment:
I don't know if `_df()` is the way to go, though: we already have a `_dfs()` method which returns children paths, so it's confusing. Also, do we consider file nodes to be dataframes, too?

I want it to be concise, so not know what to call it. We could even do just `()`, though that's hacky.